colourblind8/__init__.py

```diff
@@ -1 +1 @@
-from Colourblind8.Colourblind8 import set_deutera, set_prota, set_trita
+from Colourblind8 import Colourblind8
```

Comment:
shouldn't we be importing each method for this? instead of the package name? 
This should be
from Colourblind8 import plot_scatter

Reply:
I tried your approach and it didn't work. We should stick with the `from Colourblind8 import Colourblind8`